src/formats/yarle/utils/templates/checker-functions.ts

```diff
@@ -40,7 +41,8 @@ export const hasSourceURLInTemplate = (templateContent: string): boolean => {
 };
 export const hasAnyTagsInTemplate = (templateContent: string): boolean => {
 	return (hasItemInTemplate(TAGS, templateContent)
-		|| hasItemInTemplate(YAMLARRAYTAGS, templateContent));
```

Comment:
Should we remove the various bits of code for array tags now? It looks like they are unused?

Reply:
well, yeah, I was thinking on it, as you wish, I think the whole array tags in yaml together with its templates is currently not used. @lishid any suggestion?

Reply:
I think removing it would be best, thanks!

Reply:
@lishid removed